### ISSUE TITLE
Maple Syrup - Healthy or Just as Bad as Refined Sugar?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/health#121]
I quite like the _taste_ of [Maple Syrup](https://en.wikipedia.org/wiki/Maple_syrup) and have always considered it to be _healthier_ than refined sugar because it _appears_ to be a less refined product. But while the raw material (tree sap) is natural, there is still processing involved. 

[![image](https://user-images.githubusercontent.com/194400/70042062-2c51a200-15b6-11ea-85e1-0a88a0784f95.png)](https://youtu.be/nt5g9zcBXeE)
https://youtu.be/nt5g9zcBXeE

_However_ watching this video on how Maple Syrup is _made_, it turns out that the sap that comes from the Maple tree only contains 2-3% sugar. So the extracted sap has to be _refined_ into Maple Syrup.
They use reverse osmoses to push the sap through a membrane at high pressure.
This process takes the sugar content up to 8%.
The next step is to boil the sap in order to evaporate most of the water.
It takes 40 litres of tree sap to make 1 litre of Maple Syrup. 

[![image](https://user-images.githubusercontent.com/194400/70042809-70917200-15b7-11ea-8b31-a9967f51858f.png)](https://youtu.be/NV0RrD7ZE4Q)
https://youtu.be/NV0RrD7ZE4Q

+ https://www.healthline.com/nutrition/maple-syrup
+ https://nutritionfacts.org/video/the-healthiest-sweetener (_slow video ..._)

[![image](https://user-images.githubusercontent.com/194400/70040416-6ec5af80-15b3-11ea-8671-652463e100fa.png)](https://youtu.be/EXeSoEgnXS4)
https://youtu.be/EXeSoEgnXS4
The conclusion of the video is that Maple Syrup is _slightly_ better for you than sugar or corn syrup.
But the two sweeteners that actually have some _nutrition_ are [Date Sugar](https://en.wikipedia.org/wiki/Date_sugar) and [Molasses](https://en.wikipedia.org/wiki/Molasses).
Date sugar is the least refined of sweeteners. It's just dried dates ground up into a fine powder. It's considered a "whole food" because it still contains all the natural fibre, vitamins and minerals. (_dates are usually air-dried so they haven't been heated_) ... Molasses by contrast is the bi-product of refining sugarcane or sugar beets into sugar. It still has some nutrition (_whatever is extracted from the plant material in order to create refined sugar_) but it goes through quite a lot of processing.

# Bottom Line

Only eat Maple Syrup for the _flavour_, don't expect it to be "better for you" than sugar. 👍
